packages/base/src/util/createLinkInHead.js

```diff
@@ -12,8 +12,13 @@ const createLinkInHead = (href, attributes = {}) => {
 	Object.entries(attributes).forEach(pair => link.setAttribute(...pair));
 
 	link.href = href;
+
 	document.head.appendChild(link);
-	return link;
```

Comment:
I think we should keep the old API (`return link;`) to have it more consistent with the other similar methods (createStyle, etc...). I know I asked you to change this but after reconsidering it would be better to put the promise and events stuff directly where you use it.